page-campaigns.php

```diff
@@ -1,7 +1,9 @@
 <?php 
 	get_header();
 	$user = wp_get_current_user();
-	$subscribed = get_post_meta($user->ID, 'newsletter', true) ? get_post_meta($user->ID, 'newsletter', true) : '';
+	$subscribed = get_user_meta($user->ID, 'newsletter');
```

Comment:
This function also accepts a third parameter much like get_post_meta.  You don't have to pass it but if you decide not to you need to check if $subscribed is an array and instead of isset($subscribed[0]) sizeof($susbcribed) > 0 should be used.